apps/api-extractor/src/api/model/ApiMethod.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { ApiItemKind } from './ApiItem';
+import { ApiDeclarationMixin, IApiDeclarationMixinOptions } from '../mixins/ApiDeclarationMixin';
+import { ApiStaticMixin, IApiStaticMixinOptions } from '../mixins/ApiStaticMixin';
+import { ApiFunctionLikeMixin, IApiFunctionLikeMixinOptions } from '../mixins/ApiFunctionLikeMixin';
+import { ApiDocumentedItem, IApiDocumentedItemOptions } from './ApiDocumentedItem';
+import { ApiReleaseTagMixin, IApiReleaseTagMixinOptions } from '../mixins/ApiReleaseTagMixin';
+import { ApiResultTypeMixin, IApiResultTypeMixinOptions } from '../mixins/ApiResultTypeMixin';
+
+/** @public */
+export interface IApiMethodOptions extends
+  IApiDeclarationMixinOptions,
+  IApiFunctionLikeMixinOptions,
+  IApiReleaseTagMixinOptions,
+  IApiResultTypeMixinOptions,
+  IApiStaticMixinOptions,
+  IApiDocumentedItemOptions {
+}
+
+/** @public */
+export class ApiMethod extends ApiDeclarationMixin(ApiFunctionLikeMixin(ApiReleaseTagMixin(ApiResultTypeMixin(
```

Comment:
>ApiMethod [](start = 13, length = 9)

`ApiFunction`, since they're called functions in TS.

Reply:
A "method" is a function-like member of a class.

A "function" is a regular function that belongs to a module.


---
In reply to: [236468781](https://github.com/Microsoft/web-build-tools/pull/952#discussion_r236468781) [](ancestors = 236468781)